controllers/main.go

```diff
@@ -185,6 +206,39 @@ func (controller *MainController) getNetworkName() string {
 	return controller.params.Name
 }
 
+// agendas() returns agendas and their statuses. Fetches agenda status from
```

Comment:
Remove parentheses from function names in comments, as per [Effective Go](https://golang.org/doc/effective_go.html#commentary)